client/src/components/ImageGallery/ImageGallery.js

```diff
@@ -1,12 +1,13 @@
 import React, { useState, useCallback } from 'react';
 import { Swiper, SwiperSlide } from 'swiper/react';
 import ImageViewer from 'react-simple-image-viewer';
-
+// eslint-disable-next-line import/no-unresolved
+import 'swiper/css/bundle';
 // eslint-disable-next-line import/no-unresolved
 import 'swiper/css';
 
 // eslint-disable-next-line import/no-unresolved
```

Comment:
no need to duplicate eslint-disable-next-line import/no-unresolved 3 times,  you can use it once at the first line of the file and disable it for the whole ImageGallery.js file